src/ecdsa/ot_based_ecdsa/presign.rs

```diff
@@ -245,15 +244,12 @@ mod test {
                     keygen_out,
                     threshold,
                 },
-            );
-            assert!(protocol.is_ok());
-            let protocol = protocol.unwrap();
+            )
+            .unwrap();
```

Comment:
```suggestion
           ?;
```
you were changing it everywhere right?

Reply:
Done